helm/efk-logging/README.md

```diff
@@ -0,0 +1,30 @@
+### Setup Helm3
+```bash
+wget https://get.helm.sh/helm-v3.0.0-beta.3-linux-amd64.tar.gz
+tar -zxvf helm-v3.0.0-beta.3-linux-amd64.tar.gz
+mv linux-amd64/helm /usr/local/bin/helm3
+```
+
+### Add repos
+```bash
+helm3 repo add elastic https://helm.elastic.co
+helm3 repo add kiwigrid https://kiwigrid.github.io
+```
+
+
+### Install Helm Charts
+```bash
+helm3 install elasticsearch elastic/elasticsearch -n kube-system -f efk-logging/values-elastic.yaml
```

Comment:
Do we want these to go into the `kube-system` namespace or a separate `monitoring` namespace?

Reply:
Yes, thought about this. I think it should go into a separate namespace.

Reply:
Added a new namespace